proto/buf/alpha/audit/v1alpha1/event.proto

```diff
@@ -419,6 +422,17 @@ message PayloadRepositoryDefaultBranchChanged {
   string new_default_branch = 4;
 }
 
+message PayloadPolicyCreated {
+  // owner_id is the id of the owner of the policy.
+  string owner_id = 1;
+  // owner_name is the name of the owner of the policy.
+  string owner_name = 2;
+  // visibility is the visibility of the policy.
+  buf.alpha.registry.v1alpha1.Visibility visibility = 3;
+  // description of the policy.
+  string description = 4;
```

Comment:
Why are visibility and description set?

Reply:
It's additional useful fields in the payload. We already track visibility in other events like [the `CreateRepository` payload](https://github.com/bufbuild/buf/blob/bb812fa169445ab73d8eabad26d8f8a39f99478e/proto/buf/alpha/audit/v1alpha1/event.proto#L291-L292), and I'd be inclined to add it to [the `CreatePlugin` event](https://github.com/bufbuild/buf/blob/bb812fa169445ab73d8eabad26d8f8a39f99478e/proto/buf/alpha/audit/v1alpha1/event.proto#L436) too.

Reply:
These shared fields should be the same between these entities.